### PR TITLE
BSD support

### PR DESCRIPTION
**Pull request by heckendorfc**
I changed some commands and options in places where GNU-specific ones were used. FreeBSD has seq(1) and jot(1) but OpenBSD only has jot(1). jot isn't a drop-in replacement but it's close enough for the way it's being used in this program. Neither OS have socat(1) in the base system. The pgrep -a option doesn't exist on either system but -lf has a similar behaviour to -a piped to grep if we include both the mpv command and arguments (socket) as a regex in pgrep. BSD sed doesn't do anything useful with \n but I haven't thought of a working solution yet without introducing more dependencies.
I tested each change independently in a Linux shell and saw the expected results but I don't have a  machine with mpv to to a full test using mpvc on that platform.

### CODE REVIEW

**Comment by wildefyr** [comment]
Okay, the only issue I am seeing with this hoard of commits is [d509d38](https://github.com/wildefyr/mpvc/pull/13/commits/d509d3863f42be6cc36fce62406097ee1cd47e1) as  `pgrep -a` code is producing something like this:

`31257 mpv --really-quiet --no-audio-display --idle=yes --input-unix-socket=/tmp/mpvsocket /home/fyr/files/music/compilations/Ministry of Sound - Chilled/21 Salt Water.mp3
`

What grep is searching for is the `--input-unix-socket=/tmp/mpvsocket` bit; if grep exits with non-zero it can't find  a process with the socket, and thus needs to open a new mpv session with that socket.

Basically, revert this commit and I'll merge. To get better POSIX support, it's probably just better phrase `ps` directly.

**Comment by wildefyr** [comment]
Sorry about this, had a massive session after I saw your pull request. Just revert the pgrep commit and I'll sort out the rest.

**Comment by wildefyr** [comment]
Strike 1
